File: dotnetcore/core/Platform/Data/HistoryInfo.cs
namespace Entity.History
{
    using System;

    public class HistoryInfo
    {
        public string Data;
        public string UserId;
        public DateTime Date;

        public HistoryType Type;

        public HistoryInfo(string data, String userId, DateTime date, HistoryType type)
        {
            this.Data = data;
            this.UserId = userId;
            this.Date = date;
            this.Type = type;
        }
    }
}

File: java/core/platform/src/main/java/com/wiiee/core/platform/history/HistoryInfo.java
package com.wiiee.core.platform.history;

import com.wiiee.core.platform.constant.HistoryType;

import java.io.Serializable;
import java.time.LocalDateTime;

/**
 * Created by wiiee on 9/1/2017.
 */
public class HistoryInfo implements Serializable {
    public String data;
    public String userId;
    public LocalDateTime date;

    public HistoryType type;

    public HistoryInfo(String data, String userId, LocalDateTime date, HistoryType type) {
        this.data = data;
        this.userId = userId;
        this.date = date;
        this.type = type;
    }
}
